### ISSUE TITLE
[template] .tmpl files remain after creating a new project with template template

### ISSUE THREAD

**Issue by practicalli-johnny** [seancorfield/deps-new#47]
In a project created with the template template, i noticed several `*.tmpl` files in the root of the new project. 

My assumption is these are remnants from the original `template` template and should not have appeared in the newly created project

```
├── build.clj
├── build.tmpl
├── CHANGELOG.md
├── deps.edn
├── deps.tmpl
├── doc
│   └── intro.md
├── pom.xml
├── README.md
├── resources
...
├── template.tmpl
```

Also noted that `resources/.keep` file is in the new project, which I assume is not required for a template project, as the `resources` directory will contain other files and does not need a placeholder file.  This is a very minor issue though.

The command used to create the new project:
```shell
clojure -T:project/create :template template :name practicalli.template/service :target-dir clojure-cli-project-templates  
```

For reference, the alias used in the Clojure command is
```clojure
  :project/create
  {:replace-deps {io.github.seancorfield/deps-new {:git/tag "v0.5.0" :git/sha "48bf01e"}}
   :exec-fn      org.corfield.new/create
   :exec-args    {:template app :name practicalli/playground}}
```

**Comment by seancorfield** [comment]
Thanks. Will take a look tomorrow -- "Open Source Day" -- but it definitely looks like a bug on the surface of it.

**Comment by seancorfield** [comment]
Confirmed as a bug. Investigating.

**Comment by seancorfield** [comment]
Fixed in v0.5.1
